src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -67,17 +67,18 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		this.enableActiveCellEditOnDoubleClick();
 	}
 
-	@HostListener('document:keydown.meta.a', ['$event'])
+	@HostListener('document:keydown', ['$event'])
 	onkeydown(e) {
 		// use preventDefault() to avoid invoking the editor's select all
 		// select the active .
-		e.preventDefault();
-		document.execCommand('selectAll');
-	}
-
-	@HostListener('document:keydown.meta.z', ['$event'])
-	onUndo(e) {
-		document.execCommand('undo');
+		if ((e.ctrlKey || e.metaKey) && e.key === 'a') {
+			e.preventDefault();
```

Comment:
Could you move this to above the conditional so that it applies to both conditions?